docs/sce_file_contents.md

```diff
@@ -160,7 +168,8 @@ metadata(sce) # experiment metadata
 | `cluster_algorithm`            | The algorithm used to perform graph-based clustering of cells. Only present for `processed` objects                                                                                                                                                                                                                                                                                |
 | `cluster_weighting`            | The weighting approach used during graph-based clustering. Only present for `processed` objects                                                                                                                                                                                                                                                                                    |
 | `cluster_nn`                   | The nearest neighbor parameter value used for the graph-based clustering. Only present for `processed` objects                                                                                                                                                                                                                                                                     |
-| `celltype_methods`             | If cell type annotation was performed, a vector of the methods used for annotation. May include `"submitter"`, `"singler"` and/or `"cellassign"`. If submitter cell-type annotations are available, this metadata item will be present in all objects. Otherwise, this item will only be in `processed` objects                                                                    |
+| `celltype_methods`             | If cell type annotation was performed, a vector of the methods used for annotation. May include `"submitter"`, `"openscpca"`, `"singler"` and/or `"cellassign"`. If submitter or OpenScPCA cell-type annotations are available, this metadata item will be present in all objects. Otherwise, this item will only be in `processed` objects                                                                    |
```

Comment:
```suggestion
| `celltype_methods`             | If cell type annotation was performed, a vector of the methods used for annotation. May include `"submitter"`, `"openscpca"`, `"singler"` and/or `"cellassign"`. If submitter or OpenScPCA cell type annotations are available, this metadata item will be present in all objects. Otherwise, this item will only be in `processed` objects                                                                    |
```